src/sdk/Weapons/Projectile.js

```diff
@@ -3,35 +3,44 @@ import chebyshev from "chebyshev";
 import { Weapon } from "./Weapon";
 
 export class Projectile {
+
   /*
     This should take the player and mob object, and do chebyshev on the size of them
   */
-  constructor(damage, from, to, attackStyle, forceSWOnly) {
+  constructor(damage, from, to, attackStyle, forceSWOnly = false, image = null) {
```

Comment:
It may be better to start using an object named `options` once we start adding properties with default values (like forceSWOnly and image)
```
constructor(damage, from, to, attackStyle, options = {}) {
```

Reply:
Yeah I agree this got tricky with all the classes that construct Projectile. Will swap it over tomorrow.